x-pack/plugins/cases/server/client/cases/bulk_get.ts

```diff
@@ -88,12 +88,21 @@ const constructErrors = (
   const errors: CasesBulkGetResponse['errors'] = [];
 
   for (const soError of soBulkGetErrors) {
-    errors.push({
-      error: soError.error.error,
-      message: soError.error.message,
-      status: soError.error.statusCode,
-      caseId: soError.id,
-    });
+    if (isSODecoratedError(soError.error)) {
+      errors.push({
+        error: soError.error.output.payload.error,
+        message: soError.error.output.payload.message,
+        status: soError.error.output.statusCode,
+        caseId: soError.id,
+      });
+    } else {
+      errors.push({
+        error: soError.error.error,
+        message: soError.error.message,
+        status: soError.error.statusCode,
```

Comment:
I am a bit confused, [I'm looking at your comment here for reference](https://github.com/elastic/kibana/issues/178266#issuecomment-1985313122) and your snippet has `.status`.

Here you check for `.statusCode` which seems correct according to `SavedObjectError`.
Is the comment linked above maybe wrong?

Reply:
Sorry, what I have in my comment was wrong. I fixed it. As you said, It should be `statusCode`.